python/icra_double.py

```diff
@@ -0,0 +1,74 @@
+import random
```

Comment:
Should we get rid of icra_single and double? are they still useful outside of icra.py?